ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp

```diff
@@ -265,6 +265,18 @@ OPMapGadgetWidget::OPMapGadgetWidget(QWidget *parent) : QWidget(parent)
         }
     }
 
+    // Update the home location if it is already set:
+    if (uavo_util_mgr) {
+        bool set;
+        double LLA[3];
+        if (uavo_util_mgr->getHomeLocation(set, LLA) == 0) {
+            if (LLA[0] != 0 && LLA[1] != 0 && LLA[2] != 0)
```

Comment:
I don't think I should check it there, unless there is something I don't
understand about the use of that flag?

On Thu, Jun 4, 2015, 08:06 peabody124 <notifications@github.com> wrote:

> In ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp
> <https://github.com/TauLabs/TauLabs/pull/1639#discussion_r31729217>:
>
> > @@ -265,6 +265,18 @@ OPMapGadgetWidget::
>
> OPMapGadgetWidget(QWidget *parent) : QWidget(parent)
> >          }
> >      }
> >
> > +    // Update the home location if it is already set:
> > +    if (uavo_util_mgr) {
> > +        bool set;
> > +        double LLA[3];
> > +        if (uavo_util_mgr->getHomeLocation(set, LLA) == 0) {
> > +            if (LLA[0] != 0 && LLA[1] != 0 && LLA[2] != 0)
>
> It looks like you still aren't checking the "set" variable here. Shouldn't
> you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TauLabs/TauLabs/pull/1639/files#r31729217>.
>